src/engraving/rw/read410/tread.cpp

```diff
@@ -3686,6 +3686,14 @@ void TRead::read(StaffType* t, XmlReader& e, ReadContext&)
             e.unknown();
         }
     }
+
+    if (t->xmlName() == "stdJianpu") {
+        t->setGenClef(false);
+        t->setLines(false);
+        t->setLineDistance(Spatium(false));
+        t->setShowLedgerLines(false);
+        t->setStepOffset(false);
```

Comment:
If anything, this should be:
```suggestion
        t->setGenClef(false);
        t->setLines(0);
        t->setLineDistance(Spatium(0));
        t->setShowLedgerLines(false);
        t->setStepOffset(0);
```
But since these properties are simply irrelevant for Jianpu notation (at least, that's the understanding I get from reading the code), it might be better to just ignore these properties on Jianpu staves, instead of setting them to some value that will never be used anyway.